internal/store/elasticsearch/es.go

```diff
@@ -27,7 +27,8 @@ const (
 )
 
 type Config struct {
-	Brokers string `mapstructure:"brokers" default:"http://localhost:9200"`
+	Brokers                 string `mapstructure:"brokers" default:"http://localhost:9200"`
+	RequestTimeoutInSeconds int    `mapstructure:"request_timeout_in_seconds" default:"10"`
```

Comment:
let's use `time.Duration` instead and let's update the `compass.yaml.example` as well as a documentation
example:
https://github.com/goto/compass/blob/4495e08e81381d67d593e4be5ab3cda459c16461/internal/workermanager/worker_manager.go#L40

```suggestion
	RequestTimeout time.Duration    `mapstructure:"request_timeout" default:"10s"`
```

Reply:
If we do it this way, we will have to change the datatype and split the s and then it could be in minutes also or mili seconds also. I think better approach is to force it to be in seconds and use int value

Reply:
no need, [salt/config](https://github.com/goto/salt/blob/main/config/config.go#L142) will automatically convert this to time.Duration, its already being used on multiple configs in compass already

Reply:
done